src/modules/storage/TransactionsStorage.ts

```diff
@@ -0,0 +1,162 @@
+/*******************************************************************************
+
+    The class that insert and read the transactions into the database.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import * as sqlite from 'sqlite3';
+import { Storages } from './Storages';
+
+/**
+ * The class that insert and read the transactions into the database.
+ */
+export class TransactionsStorage extends Storages
+{
+    public createTable (callback?: any)
+    {
+        this.createTransactionsTable((err1: any) =>
+        {
+            if (err1)
+            {
+                if (callback != undefined)
+                    callback(err1);
+                return;
+            }
+        });
+    }
+
+    /**
+     * Create a table. table name is `transactions`.
+     * @param callback If provided, this function will be called when
+     * the database was finished successfully or when an error occurred.
+     * The first argument is an error object.
+     */
+    public createTransactionsTable (callback?: any)
+    {
+        let sql =
+        `CREATE TABLE IF NOT EXISTS "transactions"
+        (
+            "block_height"      INTEGER NOT NULL,
+            "tx_index"          INTEGER NOT NULL,
+            "tx_hash"           TEXT NOT NULL,
+            "type"              INTEGER NOT NULL,
+            "inputs_count"      INTEGER NOT NULL,
+            "outputs_count"     INTEGER NOT NULL,
+            PRIMARY KEY("block_height", "tx_index")
+        )`;
+        this.db.run(sql, (err: any) =>
+        {
+            if (callback != undefined)
+                callback(err);
+        });
```

Comment:
This separation is useless. `createTable` doesn't do anything more than `createTransactionsTable`. Merge them.